### PR TITLE
Luxury collection

### PR DESCRIPTION
**Pull request by irenelagno**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix(https://github.com/hlxsites/hsf-commonmoves/issues/78)

Test URLs:

Before: https://main--hsf-commonmoves--hlxsites.hlx.page/luxury-collection
-After: https://luxury-collection--hsf-commonmoves--hlxsites.hlx.page/luxury-collection

created new issue for luxury theme Property details page for https://github.com/hlxsites/hsf-commonmoves/issues/103
ttf fonts aren't available for Gotham style

### CODE REVIEW

File: blocks/columns/luxury-collection-template.css [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    margin-bottom: 25px;
+}
+
+.luxury-collection .columns.block h3 {
+    font: normal normal 700 16px/23px var(--font-family-gotham);
+    letter-spacing: 2.08px;
+    margin-bottom: 24px;
+}
+
+.luxury-collection .columns.block p{
```
Comment (bstopp) [inline blocks/columns/luxury-collection-template.css:17]: I don't know if we want to have the Lux experience be unique to it, or if we can just use the existing defintions, and just change the color and maybe the fonts.

Having a fully unique experience for lux seems over the top and too much effort to maintain in the long run?

File: blocks/columns/luxury-collection-template.css [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+}
+
+.luxury-collection .columns > div {
+    align-items: baseline;
+}
+
+.luxury-collection .columns.block img {
+    max-width: 315px;
+    height: auto;
+    float: left;
```
Comment (bstopp) [inline blocks/columns/luxury-collection-template.css:31]: Will this assumption hold true for all lux columns with an image? Will it always float on the left?

**Review by bstopp — CHANGES_REQUESTED** [review]
There seems to be a lot of unique/one-off CSS that assumes specific structure and/or organization. My guess is, this came from the live site directly.

Let's reduce the changes to the existing blocks down to theme (background/font color) and font options, and see how much of a delta that is off the current site.

The fewer variations, the better i think.

File: blocks/columns/luxury-collection-template.css [hunk 3]
```diff
@@ -0,0 +1,74 @@
+.luxury-collection .columns.block {
+    max-width: var(--normal-page-width);
+}
+
+.luxury-collection .columns.block h2 {
+    font: normal normal 300 50px/56px var(--font-family-gotham);
```
[the remark this answers is not in the document]
Comment (irenelagno) [reply blocks/columns/luxury-collection-template.css:6]: fixeded

File: blocks/columns/luxury-collection-template.css [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    letter-spacing: 0;
+    margin-bottom: 25px;
+}
+
+.luxury-collection .columns.block h3 {
+    font: normal normal 700 16px/23px var(--font-family-gotham);
+    letter-spacing: 2.08px;
```
[the remark this answers is not in the document]
Comment (irenelagno) [reply blocks/columns/luxury-collection-template.css:13]: fixed

File: styles/styles.css [hunk 5]
```diff
@@ -51,6 +51,7 @@
   --font-weight-bold: 700;
 
   /* Headings */
+  --heading-font-size-xxxxl: 50px;
```
Comment (bstopp) [inline styles/styles.css:54]: Shrink this down to xxxl; 2px shouldn't be that big of a deal for the change in experience.

**Review by bstopp — APPROVED** [review]
One minor thing but otherwise looks good!

**Review by bstopp — APPROVED** [review]

**Pull request merged**